packages/internal/src/module-utils.js

```diff
@@ -1,5 +1,6 @@
 import { readFile } from 'node:fs/promises';
 import { fileURLToPath } from 'node:url';
+import { resolve as importMetaResolve } from 'import-meta-resolve';
```

Comment:
any particular reason to use this 3rd-party `import-meta-resolve` rather than `node:module` and its `createRequire`?

Reply:
Internal consistency in our codebase: https://github.com/search?q=repo%3AAgoric%2Fagoric-sdk+importMetaResolve&type=code